app/services/airtable_placeholder_service.rb

```diff
@@ -102,67 +102,329 @@ def call(table_name)
+        'Category' => 'Shipping',
+        'Site Stage' => ['Main '],
+        'Internal Notes' => 'Check timing given COVID'
+      }, {
+        'Question' => 'I didn’t receive my Hacktoberfest 2019 T-shirt/swag.',
+        'Answer' => 'We are no longer printing 2019 T-shirts. If you did not receive your T-shirt, this means we tried to deliver multiple times unsuccessfully. We are working hard on improving the shipping process this year so that everyone receives their T-shirts. Thank you for your understanding.',
+        'Category' => 'Shipping',
+        'Site Stage' => ['Pre Launch ']
+      }, {
+        'Question' => 'Tracking for my order is not active, is my tracking number correct?',
+        'Answer' => 'You may receive a tracking email 2-3 days before tracking becomes active on DHL’s website. Try again in a few days.',
+        'Category' => 'Shipping',
+        'Site Stage' => ['Main ']
+      }, {
+        'Question' => 'How do I get started? ',
+        'Answer' => 'First, register at [hacktoberfest.digitalocean.com](https://hacktoberfest.digitalocean.com/). Then, submit at least four pull requests to any public GitHub repository that is classified with the hacktoberfest topic or has a **hacktoberfest-accepted** label on it. You can look for [open issues labeled **Hacktoberfest**](https://github.com/search?l=&o=desc&q=label%3Ahacktoberfest+state%3Aopen&s=updated&type=Issues) for inspiration. Quality contributions are encouraged! Are you maintaining a repo? Create issues or classify existing issues on your GitHub projects with **hacktoberfest** to help let new contributors know what to work on. Tag any spam or irrelevant pull requests with the **invalid** label to disqualify them.',
+        'Category' => 'General',
+        'Site Stage' => ['Main ']
+      }, {
+        'Question' => 'What is open source? ',
+        'Answer' => 'Open source refers to source code that is publicly accessible and allows anyone to inspect, modify or learn from it. Open source projects encourage collaboration and the freedom to use the software for any purpose you wish.',
+        'Category' => 'General',
+        'Site Stage' => ['Pre Launch ']
+      }, {
+        'Question' => 'When will my T-shirt be shipped? (US)',
+        'Answer' => 'You will receive an email once your T-shirt is shipped. Due to COVID-19 delivery delays, it can take up to 6-8 weeks after shipment for orders within the U.S. to be delivered.',
+        'Category' => 'Shipping',
+        'Site Stage' => ['Main '],
+        'Internal Notes' => 'Check timing given COVID'
+      }, {
+        'Question' => 'Do I get a prize for participating? ',
+        'Answer' => 'Aside from the knowledge you’ll gain (and the fun you’ll have), you’ll also receive a limited edition Hacktoberfest T-shirt for submitting 4 valid pull requests. ',
+        'Category' => 'General',
+        'Site Stage' => ['Teaser']
```

Comment:
Per #493 we do not want teaser FAQs, is that right?

Reply:
Indeed -- this placeholder data does go through that filtering, but will remove them here to avoid confusion 👍